pages/InstallationPage.ts

```diff
@@ -1,21 +1,29 @@
 import { Page, Locator } from '@playwright/test';
+import InstallationPageSelectors from '../Selector/InstallationPageSelectors';
+import MainPageSelector from '../Selector/MainPageSelectors';
 
 export default class {
 
     private readonly page : Page;
+    private readonly installationPageSelectors : InstallationPageSelectors;
 
     constructor(page: Page) {
-        this.page = page;    
+        this.page = page;
+        this.installationPageSelectors = new InstallationPageSelectors(page);  
     }
 
-    async Goto() : Promise<void> {
+    async GotoInstallationPage() : Promise<void> {
         await this.page.goto('https://playwright.dev/docs/intro');
```

Comment:
1. the goto() should be inside the util class.
2. the util function receive the url as a parameter.